crates/polars-io/src/file_cache/entry.rs

```diff
@@ -183,36 +190,74 @@ impl Inner {
 
         let data_file = finish_open(data_file_path, metadata_file);
 
+        metadata_file.set_len(0).unwrap();
```

Comment:
forgot to do this before, didn't pop up because we never wrote less bytes in an update 😅, but now we do because the TTL can be reduced